test/ApplicationData/ApplicationDataTests.cpp

```diff
@@ -237,15 +239,23 @@ namespace Test::ApplicationData::Tests
         TEST_METHOD(MachineFolderAndPath_Main_NotSupported)
         {
             winrt::hstring packageFamilyName{ Main_PackageFamilyName };
+WEX::Logging::Log::Comment(WEX::Common::String().Format(L"A1: %ls", packageFamilyName.c_str()));
```

Comment:
Nit, too late I guess - questionable indentation?

Reply:
Actually leftover debugging code. Harmless but should go. I'll clean it up in a follow up PR.